packages/ng/libraries/user/src/lib/select/searcher/user-searcher.model.ts

```diff
@@ -31,11 +31,17 @@ implements ILuApiPagedSearcherService<U> {
 		}
 	}
 	protected _get(url) {
-		return (<any>super._get(url) as Observable<ISuggestion<U>[]>)
+		return (<any>super._get(url) as Observable<ILuApiSuggestion<U>[]>)
 		.pipe(map(suggestions => suggestions.map(s => s.item)));
 	}
 	protected _clueFilter(clue) {
 		const urlSafeClue = clue.split(' ').map(c => encodeURIComponent(c)).join(',');
 		return `clue=${urlSafeClue}`;
 	}
+	getMe(): Observable<ILuUser> {

```

Comment:
Why don't we just pass a new input to user-select ? Most of the time (if not all the time), apps know who is connected. That way, you don't have to make 2 API calls per user-picker and you can easily control whether or not to display "me" option.

Reply:
i wanted the component to be self-sufficient and i'm never too shy about 2 http calls.

these calls are fired when the select opens, not when it instantiates, so thes 2 calls wont be added to the myriad that fire when a screen bootstraps, so i dont think it's _that_ impactfull

i still need to check if `me` is a possible option in the user-select with the filters that are asked so 1 of the api calls is still necessary (for example if the user-select is on the population of user you can approve expenses of)

> you can easily control

as i have stated a the previous comment i'd rather not people _control_ when it appears. either its a possible choice and it appears first, or its not and it doesn't. that way the UX for the end user is very predictible

i might add a `LU_USER_ME` injection token in the future if more component seem to need it to avoid - when provided - the first call to `/api/v3/users/me`

Reply:
👌